src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/DeclarationTests.cs

```diff
@@ -228,8 +231,12 @@ .maxstack  4
   IL_0037:  call       ""string Microsoft.VisualStudio.Debugger.Clr.IntrinsicMethods.GetVariableAddress<string>(string)""
   IL_003c:  ldloc.2
   IL_003d:  stind.ref
-  IL_003e:  ret
-}");
+  IL_003e:  ldloc.1
+  IL_003f:  ldloc.2
+  IL_0040:  newobj     ""System.ValueTuple<int, string>..ctor(int, string)""
```

Comment:
This is for the result of `CompileExpression`... it is the result of the expression. The expression compiler always produces the result of the expression for the consumer to use. This change is caused by the fact that the result of the deconstruction is now a tuple instead of void.